examples/CampaignManagement/AddCompleteCampaignsUsingMutateJob.php

```diff
@@ -137,17 +137,53 @@ public static function runExample(GoogleAdsClient $googleAdsClient, $customerId)
     {
         $mutateJobServiceClient = $googleAdsClient->getMutateJobServiceClient();
 
-        // Creates a new mutate job.
+        $mutateJobResourceName = self::createMutateJob($mutateJobServiceClient, $customerId);
+        self::addAllMutateJobOperations(
+            $mutateJobServiceClient,
+            $customerId,
+            $mutateJobResourceName
+        );
+        $operationResponse = self::runMutateJob($mutateJobServiceClient, $mutateJobResourceName);
+        self::pollMutateJob($operationResponse);
+        self::printResults($mutateJobServiceClient, $mutateJobResourceName);
+    }
+
+    /**
+     * Creates a new mutate job for the specified customer ID.
+     *
+     * @param MutateJobServiceClient $mutateJobServiceClient the mutate job service client
+     * @param int $customerId the client customer ID
+     * @return string the created mutate job's resource name
```

Comment:
Nit: I think this would sound better as "the resource name of the created mutate job"

Reply:
Done.